libs/gi/sheets/src/Artifacts/ScrollOfTheHeroOfCinderCity/index.tsx

```diff
@@ -27,24 +27,23 @@ const condReacts = objKeyMap(allElementKeys, (ele) =>
   condReadNode(condReactPaths[ele])
 )
 
-const condReactNodes = objKeyMap(allElementKeys, (triggerEle) =>
-  greaterEq(
-    input.artSet.ScrollOfTheHeroOfCinderCity,
-    4,
-    equal(condReacts[triggerEle], triggerEle, 1)
-  )
+const reactNodeOnCount = sum(
+  ...allElementKeys.map((ele) => equal(condReacts[ele], ele, 1))
 )
-const reactNodeOnCount = sum(...Object.values(condReactNodes))
 
 const condReactBuffs = objKeyValMap(allElementKeys, (ele) => [
   `${ele}_dmg_`,
-  threshold(
-    equal(input.charEle, ele, 1),
-    1,
-    // buffing self elemental damage, check if any of the reactions are enabled
-    greaterEq(reactNodeOnCount, 1, 0.12),
-    // buffing other elemental damage, check if that particular conditional is enabled
-    equal(condReactNodes[ele], 1, 0.12)
+  greaterEq(
+    input.artSet.ScrollOfTheHeroOfCinderCity,
+    4,
+    threshold(
```

Comment:
`threshold(equal(v1, v2, 1), 1, v3, v4)` can be replaced with `compareEq(v1, v2, v3, v4)`